tasties_app/models.py

```diff
@@ -105,6 +106,25 @@ class Comment(models.Model):
     publication_date = models.DateField(default=timezone.now)
     comment_text = models.CharField(max_length=512, validators=[MinLengthValidator(1)])
 
+    def __str__(self):
+        try:
+            a_id = str(self.author_id)
+            comment = self.comment_text
+            date1 = str(self.publication_date)
+            string = a_id+' '+comment+' '+date1
+            return string
+        except ValidationError as e:
+            raise (e)
+
+    # function to add comment with length check
+    def add_comment(self, text):
+        if len(text) < 1:
+            raise ("len the text is illegal")
```

Comment:
I suggest wherever you raise an error with an error you log I would suggest a ValidationError error

Reply:
I think is the same. I want just to stop the code to know the costumer the comment is empty .